kitsune/wiki/views.py

```diff
@@ -525,6 +525,20 @@ def review_revision(request, document_slug, revision_id):
     revision_contributors = list(set(
         based_on_revs.values_list('creator__username', flat=True)))
 
+    # Get Unreviewed Revisions
+    unreviewed_revision = list(Revision.objects.filter(
```

Comment:
Also, generally calling `list()` on an ORM query is a sign that something isn't quite right. If you don't do the `if rev_in unreviewed_revision` thing below, you won't need to do `list()` here. That will be more efficient, as it will only get the first 5 from the DB, and not the entire list of every revision that matches these criteria.